guvnor-ala/guvnor-ala-build-maven/src/main/java/org/guvnor/ala/build/maven/config/MavenBuildConfig.java

```diff
@@ -34,14 +34,14 @@
     */
     default List<String> getGoals() {
         final List<String> result = new ArrayList<>();
-        result.add( "package" );
+        result.add("package");
         return result;
     }
 
     default Properties getProperties() {
         final Properties result = new Properties();
-        result.setProperty("failIfNoTests", "false" );
+        result.setProperty("failIfNoTests",
+                           "false");
```

Comment:
"To be honest I'm completely fed up with talking about code formatting." +1
Happy to see this PR being merged and seeing more consistency in guvnor/wb projects and it being in general (but not individually as pointed out by Jiri) more consistent with the original Sun Java conventions and the JBoss extension of those. **optaplanner-wb should also be formatted as such to be consistent with the rest of the wb projects.** That being said, I am not keen on reformatting the optaplanner core engine repository like this, as that one does follow the Sun Java conventions and JBoss extensions of those, until I have time to take a closer look to our dialect of that. Bikeshedding to be continued, I am sure :)